OrbitGl/App.cpp

```diff
@@ -155,25 +154,26 @@ void OrbitApp::OnCaptureStarted(ProcessData&& process,
 
 void OrbitApp::OnCaptureComplete() {
   capture_data_.FilterBrokenCallstacks();
-  SamplingProfiler sampling_profiler(*capture_data_.GetCallstackData(), capture_data_);
-  capture_data_.set_sampling_profiler(sampling_profiler);
-  main_thread_executor_->Schedule(
-      [this, sampling_profiler = std::move(sampling_profiler)]() mutable {
-        RefreshCaptureView();
+  main_thread_executor_->Schedule([this]() {
+    SamplingProfiler sampling_profiler(
```

Comment:
Can you trie to move this one line above again to do this on the capture thread?

Reply:
done